scripts/CEE/retrieve-pdbs/script.sh

```diff
@@ -0,0 +1,78 @@
+#!/bin/bash
+
+set -e
+set -o nounset
+set -o pipefail
+
+# Function to retrieve all namespaces starting with openshift
+get_openshift_namespaces() {
+    oc get namespaces -o json | jq -r '.items[].metadata.name' | grep '^openshift'
+}
+
+# Function to retrieve and display PDBs for a given namespace
+get_and_display_pdbs() {
+    local namespace="$1"
+    echo "---------------------------------"
+    echo "PDBs in namespace: $namespace"
+    echo ""
+    oc get poddisruptionbudget -n "$namespace"
+}
+
+# Check if the 'namespace' variable is provided
+if [ -z "${namespace:-}" ]; then
+    echo "No 'namespace' provided. Retrieving PDBs from all openshift namespaces."
+    openshift_namespaces=( $(get_openshift_namespaces) )
+
+    # Loop through each openshift namespace and display PDBs
+    for namespace in "${openshift_namespaces[@]}"; do
```

Comment:
Use a for loop is kind of time consuming as well as api call consuming.
Do we have the special use case to split the namespaces like this?
Or can we just simplify the script to only retrieve the PDB for all namespaces?